### PR TITLE
Toolchain package example

### PR DESCRIPTION
**Pull request by czoido**
Related to https://github.com/conan-io/docs/pull/3625

### CODE REVIEW

File: tutorial/creating_packages/other_packages/toolchain_packages/run_example.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    generators_folder = "Release/generators"
+
+    with chdir("consumer/build"):
+        command = []
+        # in the conanfile.py we only add CMake as tool_require in Linux
+        command.append(f". {generators_folder}/conanbuild.sh")
+        command.append(f"cmake .. -DCMAKE_TOOLCHAIN_FILE={generators_folder}/conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release")
+        command.append("cmake --build .")
+        command.append(f". {generators_folder}/deactivate_conanbuild.sh")
+        run(" && ".join(command))
```
Comment (memsharded) [inline tutorial/creating_packages/other_packages/toolchain_packages/run_example.py:23]: Worth trying to go for the ``--preset`` feature with env? Would be a great user experience

File: tutorial/creating_packages/other_packages/toolchain_packages/toolchain/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import os
+from conan import ConanFile
+from conan.tools.files import get, copy, download
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+
+# conan create toolchain.py -pr:b=default -pr:h=./profiles/raspberry-64 --build-require -c:b tools.files.download:verify=False
```
Comment (memsharded) [inline tutorial/creating_packages/other_packages/toolchain_packages/toolchain/conanfile.py:7]: Is ``tools.files.download:verify=False`` necessary? Only for us?

**Review by memsharded — APPROVED** [review]
This is great

File: tutorial/creating_packages/other_packages/toolchain_packages/toolchain/conanfile.py (hunk 2 above)
Comment (czoido) [reply tutorial/creating_packages/other_packages/toolchain_packages/toolchain/conanfile.py:7]: Sorry, that was I left that as a comment for myself and forgot to remove it. Removing it.

File: tutorial/creating_packages/other_packages/toolchain_packages/run_example.py (hunk 1 above)
Comment (czoido) [reply tutorial/creating_packages/other_packages/toolchain_packages/run_example.py:23]: I'll add the presets one too 👍

File: tutorial/creating_packages/other_packages/toolchain_packages/toolchain/test_package/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+from io import StringIO
+from conan import ConanFile
+from conan.tools.cmake import CMake, cmake_layout
+import os
+
+
+class TestPackgeConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeToolchain", "VirtualBuildEnv"
+    test_type = "explicit"
```
Comment (memsharded) [inline tutorial/creating_packages/other_packages/toolchain_packages/toolchain/test_package/conanfile.py:10]:
```suggestion
```

**Review by memsharded — APPROVED** [review]

File: tutorial/creating_packages/other_packages/toolchain_packages/toolchain/test_package/conanfile.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+from io import StringIO
+from conan import ConanFile
+from conan.tools.cmake import CMake, cmake_layout
+import os
+
+
+class TestPackgeConan(ConanFile):
```
Comment (czoido) [inline tutorial/creating_packages/other_packages/toolchain_packages/toolchain/test_package/conanfile.py:7]:
```suggestion
class TestPackageConan(ConanFile):
```

**Pull request merged**